docs/how-to-play-9c-on-linux-and-mac.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Play 9c on x86 macOS / Linux using CI Artifacts (UNOFFICIAL SUPPORT)
+---
+# DISCLAMER
+THIS UNOFFICIAL METHOD DOES NOT GUARANTEE THE NORMAL OPERATION OF THE SOFTWARE (INCLUDING LAUNCHER AND GAME CLIENT) AND WE DOES NOT PROVIDE OFFICIAL TECHNICAL SUPPORT. OUR PRIMARY ISSUE PRIORITY IS ALWAYS WINDOWS FIRST. AND THESE TARGETS ARE USED FOR INTERNAL TEST ONLY. USE AT YOUR OWN RISKS. THIS EXPLANATION WILL PRESUME YOU KNOW WHAT YOU'RE DOING.
+
+# 1. Get Artifact
+
+## [1. Open 9c-launcher CircleCI Pipelines page](https://app.circleci.com/pipelines/github/planetarium/9c-launcher?branch=main)
+
+
+[Here.](https://app.circleci.com/pipelines/github/planetarium/9c-launcher?branch=main)
+
+## 2. Click Workflow "build", and click Succeed (Green) "build-{your OS}" 
+
+
+## 3. Click Artifact
+
+
+
+## 4. Download and Decompress Artifact
+
+
+
+# 2. Run on Linux
```

Comment:
I think `# 2` should be more general title and each OS guideline should belongs to the `# 2` section.

Reply:
Yes, you're right, actually I think this document's levels needs to be totally reordered. will be fixed.